keepalived/cluster.sls

```diff
@@ -43,6 +43,9 @@ keepalived_{{ instance_name }}_notify:
 
 {%- for name, script in cluster.get('vrrp_scripts', {}).iteritems() %}
 keepalived_vrrp_script_{{ name }}:
+{%- if script.name is defined %}
+{%- set name=script.name %}
```

Comment:
Counted, it's still using the same script. So iterates multiple times and setting up the same file.

On keepliaved it will name the vrrp as "check mysql port" or "check haproxy port" and call the `vrrp_script_check_port.sh ARG1 ARG2` where ARG* differ.

on the line you pointed we want to create such file on filesystem (from template) but the file name must rename generic for [mysql port, haproxy port check] thus will update the name. On the same hand for different use-cases we want to have script created on fs to have a name as a pillar key on model, thus if no inner name is defined, the vrrp_sripts: key aka "name" is used.



vrrp_script_check_mysql.sh -> will remain the same, but